man/man5/zpool-features.5

```diff
@@ -14,6 +11,10 @@
 .\" CDDL HEADER, with the fields enclosed by brackets "[]" replaced with your
 .\" own identifying information:
 .\" Portions Copyright [yyyy] [name of copyright owner]
+.\" Copyright (c) 2012, 2018 Delphix. All rights reserved.
```

Comment:
you don,t like them, eh? :D

Reply:
I am starting to get an issue with them **by** now.